bot/resources/tags/defaultdict.md

```diff
@@ -0,0 +1,20 @@
+**[`collections.defaultdict`](https://docs.python.org/3/library/collections.html#collections.defaultdict)**
+
+The Python `defaultdict` type behaves almost exactly like a regular Python dictionary, but if you try to access or modify a missing key, the `defaultdict` will automatically create the key and generate a default value for it. 
```

Comment:
The key is not being created; the key is being entered into the dict and a default value is created.

I think everyone will know what you mean by "access or modify a missing key", though one could argue that it is the *value* that is being accessed or modified via a certain key. Though it's probably not worth adding extra words to uphold that distinction.